src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -47,6 +49,7 @@ function stripNewlines(str: string): string {
 
 export interface CodeActionShowOptions {
 	readonly includeDisabledActions: boolean;
+	readonly fromLightbulb: boolean;
```

Comment:
Instead of introducing a new field here, you should be able to check `CodeActionTrigger.type`:

https://github.com/microsoft/vscode/blob/6e1ea79174dd791a0db49c96a9b986ae44823306/src/vs/editor/common/languages.ts#L675